subsys/ipc/ipc_service/backends/ipc_icbmsg.c

```diff
@@ -110,6 +110,12 @@ LOG_MODULE_REGISTER(ipc_icbmsg,
 /** Registered endpoints count mask in flags. */
 #define FLAG_EPT_COUNT_MASK 0xFFFF
 
+/** Workqueue stack size for bounding processing. */
+#define EP_BOUND_WORK_Q_STACK_SIZE (512U)
```

Comment:
How the stack size of this workqueue was determined? This could be a memory waste

Reply:
It was an arbitrary decision without any profiling on my side. I didn't want to make this parameter too low as it increases the risk of running into stack overflow issues.

I still treat this change as a temporary solution and I briefly talked with the author`s module - @doki-nordic, who suggested that the long-term solution requires reusing a dedicated work queue from the ICMsg module. He plans to implement this part later:

https://github.com/nrfconnect/sdk-zephyr/pull/1676#pullrequestreview-2040493888

The motivation for applying this change right now is to unlock development for the nRF54H20 target in one of our applications - nRF Desktop, which is based on the App Event Manager module. This module works in the System Workqueue context and fails to initialize Bluetooth (`bt_enabled) due to this IPC issue.

Reply:
I would add a comment that this stack size can be optimized. Just in case the implementation of the permanent solution takes longer than expected

Reply:
Sure, I added a comment in the last patch. You can see the force push diff using this link:

https://github.com/zephyrproject-rtos/zephyr/compare/8b2ccf06ac04bc96b4df78d94e8d4741f1f6c560..5712362287c1da2a60d9914a243817251b3f5c18